Tools/buildbot/test.bat

```diff
@@ -37,8 +37,12 @@ set TEMP_ARGS=--temp %REMOTE_PYTHON_DIR%temp
 
 set rt_args=%rt_opts% %dashU% -rwW --slowest --timeout=1200 --fail-env-changed %regrtest_args% %TEMP_ARGS%
 ssh %SSH_SERVER% "set TEMP=%REMOTE_PYTHON_DIR%temp& %REMOTE_PYTHON_DIR%PCbuild\rt.bat" %rt_args%
+set ERR=%ERRORLEVEL%
+echo where scp
+where scp
+echo scp %SSH_SERVER%:"%REMOTE_PYTHON_DIR%test-results.xml" "%PYTHON_SOURCE%\test-results.xml"
 scp %SSH_SERVER%:"%REMOTE_PYTHON_DIR%test-results.xml" "%PYTHON_SOURCE%\test-results.xml"
```

Comment:
"%PYTHON_SOURCE%\test-results.xml" should be "%PYTHON_SOURCE%test-results.xml"
I don't know if the extra slash will matter to scp or not without testing.